install/tools/setupIstioVM.sh

```diff
@@ -57,14 +57,12 @@ function istioNetworkInit() {
 function istioInstall() {
   echo "*** Fetching istio packages..."
   # Current URL for the debian files artifacts. Will be replaced by a proper apt repo.
-  curl -f -L ${PILOT_DEBIAN_URL}/istio-agent.deb > ${ISTIO_STAGING}/istio-agent.deb
-  curl -f -L ${AUTH_DEBIAN_URL}/istio-auth-node-agent.deb > ${ISTIO_STAGING}/istio-auth-node-agent.deb
-  curl -f -L ${PROXY_DEBIAN_URL}/istio-proxy.deb > ${ISTIO_STAGING}/istio-proxy.deb
+  rm -f istio-sidecar.deb
+  echo curl -f -L ${PILOT_DEBIAN_URL}/istio-sidecar.deb > ${ISTIO_STAGING}/istio-sidecar.deb
```

Comment:
do we need the echo?

Reply:
Well, I needed it - since it was not clear what the curl was doing. I don't think it hurts, but it may help. The actual file it attempts to download doesn't actually work, I had to manually patch the version file.